testnet/stacks-node/src/tests/signer/v0.rs

```diff
@@ -1760,6 +1760,7 @@ fn miner_forking() {
             config.node.pox_sync_sample_secs = 30;
             config.burnchain.pox_reward_length = Some(max_sortitions as u32);
             config.miner.block_commit_delay = Duration::from_secs(0);
+            config.miner.tenure_cost_limit_per_block_percentage = Some(100);
```

Comment:
Won't this cause an error?

Reply:
Nope. It doesn't parse using the miner config file parsing. but you are right that I need toa llow 100 in the miner config parsing check.